server.js

```diff
@@ -1,33 +1,73 @@
-import express from 'express'
+import express, { response } from 'express'
```

Comment:
`response` should be a part of `express`, did you have trouble with using it without specifying it?

Reply:
I can't remember adding this one, I think vscode adds things by itself sometimes 😝